coderd/prometheusmetrics/prometheusmetrics.go

```diff
@@ -516,6 +516,31 @@ func AgentStats(ctx context.Context, logger slog.Logger, registerer prometheus.R
 	}, nil
 }
 
+// Experiments registers a metric which indicates whether each experiment is enabled or not.
+func Experiments(_ slog.Logger, registerer prometheus.Registerer, exps []string, all codersdk.Experiments) error {
```

Comment:
Logger is unused, consider removing unless you are planning to use it in a follow up PR?

Also noticed the mixed use of `[]string` and `codersdk.Experiments`, these should be interchangeable, would be clearer to use the same type for both.

Reply:
I added the logger to keep consistency with the method signatures of the other metric functions, but yeah I can remove it.

> Also noticed the mixed use of []string and codersdk.Experiments, these should be interchangeable, would be clearer to use the same type for both.
Yeah it did feel a bit uncomfortable. OK, I'll do that, thanks.